script/cibuild

```diff
@@ -2,3 +2,4 @@
 set -e # halt script on error
 
 bundle exec htmlproofer .
+eslint js/*
```

Comment:
This is a pretty open wildcard, but I don't expect we will store non-`.js` files there.